### DISCUSSION TITLE
How do you link both your main email and school email using Killcurly

### DISCUSSION THREAD

**Discussion by videoconsole** [3kh0/ext-remover#592] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so when i use killcurly i dont use my school email i use my main because restarting my chromebook with killcurly messes with the blocks and blocks more stuff instead so i just use my main for whenever i want to do random stuff

but one time i was on my main and suddenly my school account was linked also and i dont know how that worked but i was wondering if anyone knows how to replicate it.

**Comment by S-PScripts** [comment, marked as answer]
n
